src/azure-cli-core/azure/cli/core/tests/test_help.py

```diff
@@ -610,7 +610,7 @@ def test_example_unsupported_profiles(self):
         ex_dict = {
             'summary': "Example to test supported-profiles",
             'command': "Az foo bar",
-            'unsupported-profiles': " 2017-03-09-profile, 2018-03-01-hybrid, latest "
```

Comment:
@adewaleo please take a look at this. Even this test should not incur maintenance burden because a new profile is added.

Reply:
I can update the test to not require hard-coding / updating profile names